app/models/concerns/push_updates_for_message.rb

```diff
@@ -92,7 +92,7 @@ def state_for_user(user_id)
   def app_name_for_user(user_id)
     obj = self.related_object
     created_by_id = obj.try(:created_by_id) || obj.try(:offer).try(:created_by_id)
-    if object_class == 'Order'
+    if ['Order', 'Package'].include?(object_class)
       (created_by_id == user_id) ? BROWSE_APP : STOCK_APP
     else # Offer/Item
       (created_by_id == user_id) ? DONOR_APP : ADMIN_APP
```

Comment:
I'm not sure this logic is going to work for Package here. We basically want to alert the users on the stock app only

```suggestion
    if ['Order'].include?(object_class)
      (created_by_id == user_id) ? BROWSE_APP : STOCK_APP
    elsif ['Offer', 'Item'].include?(object_class)
      (created_by_id == user_id) ? DONOR_APP : ADMIN_APP
    elsif ['Package'].include?(object_class)
      STOCK_APP
```

Reply:
ah right, my bad. Thanks for pointing out this @steveyken